save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -50,13 +51,26 @@ class Contest(
     /**
      * @return set of testSuiteIds
      */
-    fun getTestSuiteIds() = testSuiteIds.split(",")
+    fun getTestSuiteIds() = testSuiteIds.split(DELIMITER)
         .mapNotNull {
             it.toLongOrNull()
         }
         .toSet()
 
+    private fun isTestSuiteIdsValid() = testSuiteIds.isBlank() || testSuiteIds.all { it.isDigit() || it == DELIMITER }
+
+    private fun isDateRangeValid() = startTime != null && endTime != null && (startTime as LocalDateTime) < endTime
+
+    /**
+     * Validate contest data
+     *
+     * @return true if contest data is valid, false otherwise
+     */
+    fun isValid() = isNameValid(name) && isTestSuiteIdsValid() && isDateRangeValid()
+
     companion object {
+        private const val DELIMITER = ','
```

Comment:
suggestion to reuse `com.saveourtool.save.entities.Execution#TEST_SUITE_IDS_DELIMITER`

Reply:
Probably, we can even unify all such delimiters and constans in some `Constants.kt` in `save-cloud-common`
Now all of them are scattered in the project

Reply:
Agree, will do it

Reply:
For now it seems that all the delimiters we use are commas, so I moved it Constants.kt to be `DELIMITER`